content/_changelogs/9.0.0.md

```diff
@@ -0,0 +1,65 @@
+## 9.0.0
+
+_Released 11/08/2021_
+
+**Breaking Changes:**
+
+- The `nodeVersion` configuration option now defaults to `system`. The behavior
+  of the `system` option has changed to always use the Node.js binary/version
+  that launched Cypress. If Cypress was not launched via the terminal, Cypress
+  will use the `bundled` Node.js version. This could change the behavior of code
+  within your `pluginsFile` since it may be run in your `system` Node.js
+  version. Addresses
+  [#18684](https://github.com/cypress-io/cypress/issues/18684).
+- Windows 32-bit is no longer supported in Cypress. Addresses
+  [#17962](https://github.com/cypress-io/cypress/issues/17962).
+- An error will now be thrown if an invalid value is passed to `Cypress.config`.
+  Previously invalid values were ignored and could cause unexpected behavior.
+  Addresses [#18589](https://github.com/cypress-io/cypress/pull/18589).
+- `cy.contains()` no longer yields the `<body>` element when it matches the
+  content of `<script>` or `<style>` tags. Addresses
+  [#14861](https://github.com/cypress-io/cypress/issues/14861).
+- Attempting to add an existing built-in Cypress command using
+  `Cypress.Commands.add()` will now throw an error, indicating that
+  `Cypress.Commands.overwrite()` should be used instead to overwrite the
+  behavior of existing commands. Addresses
+  [#18572](https://github.com/cypress-io/cypress/issues/18572).
+- Custom command implementations are now typed based on the declared custom
+  chainables. Addresses
+  [#17496](https://github.com/cypress-io/cypress/issues/17496).
+- The bundled Node.js version was upgraded from `14.17.0` to `16.5.0`. This
+  could change the behavior of code within the `pluginsFile` when using the
+  bundled Node.js version of Cypress. Addressed in
+  [#18317](https://github.com/cypress-io/cypress/pull/18317).
+
+**Deprecations:**
+
+- The `nodeVersion` configuration option has been deprecated and will be removed
+  in a future release.
+
+**Features:**
+
+- When `null` is passed as the encoding to `cy.readFile()`, the file is treated
+  as binary and read as a Buffer. Similarly, `null` passed as the encoding to
+  `cy.writeFile()` allows direct writing of buffers. If the encoding is
+  unspecified, the default remains `utf8`, matching the current behavior.
+  Addresses [#18534](https://github.com/cypress-io/cypress/issues/18534).
```

Comment:
Should we also call out that this impacts fixture?

Based on: https://github.com/cypress-io/cypress-documentation/pull/4172

Reply:
Yes! Thanks.

Reply:
This still needs to be addressed, right? What should be added concerning fixtures?

Reply:
I thought that I changed this.

Reply:
my bad, committing now.